app/middlewares/schema_validation.js

```diff
@@ -0,0 +1,12 @@
+const { checkSchema, validationResult } = require('express-validator/check');
+
+const errors = require('../errors');
+
+const checkValidationResult = (req, res, next) =>
+  validationResult(req).isEmpty()
+    ? next()
+    : next(errors.badRequestError(validationResult(req).errors[0].msg));
```

Comment:
what if there is more than one error?

Reply:
Should i make a response with all the error messages?

Reply:
yes